.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         make -C test
     - name: Test run
       run: |
-        source /tmp/${HPCX_PKG_NAME}/hpcx-init.sh; hpcx_load
+        source /tmp/${HPCX_PKG_NAME}/hpcx-mt-init.sh; hpcx_load
```

Comment:
should we add new tests to CI?

Reply:
we added it last time in order to try and pass the checks... should i remove it and see if it passes?

Reply:
@Sergei-Lebedev ?

Reply:
I mean add test_mpi_mt to CI

Reply:
added